apiaccess.py

```diff
@@ -37,10 +60,14 @@ def call_endpoint(endpoint, auth_string, request_spec, case_id,
     if request_spec:
         resp = requests.post(
             url,
+            params=_flatten_params_dict(params),
```

Comment:
request.post or request.get set URL parameters from dictionary, so the correct dictionary here would be: `{"age.value": x, "age.unit": y}`, but in different endpoints this information about age is send via json (in post body), where structure is `{"age": {"value": x, "unit":y}}`. Alternatively we can omit this  `_fltten_params_dict` function and pass sth like ```call_endpoint(..., params={"age.value": age["value"], "age.unit": age["unit"]}``` What do you think?

Reply:
I think it would be better to do this directly using `age.unit` etc. This piece of code is intended to be a demo showing that our APIs are simple :sweat_smile: so I'd rather avoid suggesting potential users that you really need to implement a dict-flattening routine yourself to use the API.

Reply:
okay, I've simplified it :)

Reply:
Maybe squash my 4 commits into one?